com.github.taburineagle.NeewerLite-Python.metainfo.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Typically, you should maintain it in your repo.

Reply:
It the goal we have a upstream PR that shoud be merged soon i hope